Backpack/FlareView/Classes/BPKFlareView.h

```diff
@@ -62,5 +62,7 @@ IB_DESIGNABLE @interface BPKFlareView : UIView
  */
 - (instancetype)initWithFrame:(CGRect)frame NS_DESIGNATED_INITIALIZER;
 
+- (CGSize)flareSize;
```

Comment:
Want this to be undocumented as we don't want consumers to override it

Reply:
Since the aspect ratio is always fixed would it be better to expose a way to set one of `height` or `width` and have the other one be calculated? This would mean it's impossible to render the flare distorted at least